test/fixtures/auth0-oidc-example-profile.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I dont think this is a OIDC profile right?

Reply:
indeed.. I just put the `sub` there 😁